internal/deviations/deviations.go

```diff
@@ -66,9 +66,23 @@ package deviations
 import (
 	"flag"
 
+	log "github.com/golang/glog"
+	"github.com/openconfig/featureprofiles/internal/metadata"
+	mpb "github.com/openconfig/featureprofiles/proto/metadata_go_proto"
 	"github.com/openconfig/ondatra"
 )
 
+// resolveDutDeviations returns the deviations for the specified dut. If no duts match `nil` is returned.
+func resolveDUTDeviations(dut *ondatra.DUTDevice) *mpb.Metadata_Deviations {
```

Comment:
Sorry, the verb "resolve" is bothering me a bit here, because I'm not sure there's much resolution going on. I think "lookup" may be more appropriate?

Reply:
done